lib/vagrant-google/action/run_instance.rb

```diff
@@ -113,11 +115,21 @@ def call(env) # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
             network = "global/networks/default"
           end
 
+
```

Comment:
This whole block should not be necessary as we've implemented the change in `fog-google`